dmscripts/upload_dos_opportunities_email_list.py

```diff
@@ -31,14 +27,14 @@ def main(data_api_client, mailchimp_client, lot_data, logger):
         "{} emails have been found for lot {}".format(len(emails), lot_data["lot_slug"])
     )
 
-    existing_mailchimp_emails = mailchimp_client.get_email_addresses_from_list(lot_data["list_id"])
+    existing_mailchimp_emails = list(mailchimp_client.get_email_addresses_from_list(lot_data["list_id"]))
```

Comment:
Do you actually need to convert it to a list here?..

Reply:
Sadly it needs to be all slurped into memory at a later point to do a set difference with the known DM-side emails, so I figured it was easiest to just do it upfront than a couple lines later.

Reply:
We also iterate over the list twice which makes it harder to use a generator. 🙈